iambic/plugins/v0_1_0/aws/models.py

```diff
@@ -299,6 +306,16 @@ class AWSAccount(ProviderChild, BaseAWSAccountAndOrgModel):
         description="The role arn to assume into when making calls to the account",
         exclude=True,
     )
+    organization_account: bool = Field(
+        False, description="if this is an organization account"
+    )
+    organization: Optional[AWSOrganization] = Field(
```

Comment:
It's not a blocker but at some point it may be worth circling back and removing `organization` and `aws_config` to prevent accidental circular refs. 

We could do something like adding the additional required org attributes to the AWSAccount object and have some type of org directory on the org AWSAccount. We're currently doing something similar with CloudUMI.

Reply:
Indeed the problem is you are not able to know if your account should be used to `execute` any operation over aws-organizations. You need the config to parse the account names to id's.